src/pages/Home/components/LeagueSelector/style.scss

```
@@ -124,21 +131,52 @@
     &::placeholder {
       color: #bebebe;
       font-size: 0.8rem;
     }
 
     option {
-      color: #bebebe;
+      color: #333;
+      background-color: #fff;
+      padding: 8px;
+
+      max-height: 150px;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)